test/fixtures/browser/features/steps/browser_steps.rb

```diff
@@ -1,6 +1,35 @@
+def get_document_server_url
+  if Maze.config.aws_public_ip
+    "http://#{Maze.public_document_server_address}"
+  else
+    "http://#{get_private_hostname}:#{Maze.config.document_server_port}"
+  end
```

Comment:
Presumably we'd need to do this in the SDKs, so can Maze Runner provide an API?

Currently we're doing this:

```ruby
maze_uri = URI("http://#{Maze.config.bind_address}:#{Maze.config.port}")
fixtures_uri = URI("http://#{Maze.config.document_server_bind_address}:#{Maze.config.document_server_port}")
```

Reply:
That would seem to make sense, but I think I'll add it separately when I come to move the JS/perf pipelines over.